src/backends/backend/triton_backend_manager.cc

```diff
@@ -327,15 +327,29 @@ TRITONBACKEND_BackendSetState(TRITONBACKEND_Backend* backend, void* state)
 //
 // TritonBackendManager
 //
+
+TritonBackendManager&
+TritonBackendManager::Instance()
```

Comment:
Instead of breaking the encapsulation of the class by exposing these internals, just add a BackendState() method that fills a map from backend-name to a record of state information (path and config).